Sources/ZcashLightClientKit/Synchronizer/SDKSynchronizer.swift

```diff
@@ -302,6 +309,8 @@ public class SDKSynchronizer: Synchronizer {
         memo: Memo,
         shieldingThreshold: Zatoshi
     ) async throws -> PendingTransactionEntity {
+        if !latestState.syncStatus.isPrepared { throw SynchronizerError.notPrepared }
```

Comment:
we could reduce this with a helper function that coneys meaning.
```suggestion
       throwIfUnprepared()
```

Reply:
Done